Client/src/Components/product-reviews/star-rating/RecommendProductPercentage.jsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+// Takes in recommendList, a boolean array where true means the reviewer recommends the product
+// and false if the reviewer does not recommend the product. It returns the statement:
+// "X% of reviews recommend this product."
+function RecommendProductPercentage({ list }) {
+  // countPercentage calculates the percentage of trues in recommendList.
+  function countPercentage() {
+    let recommendCount = 0;
+    list.forEach((recommend) => {
+      if (recommend) {
+        recommendCount += 1;
+      }
+    });
+    return (recommendCount / list.length) * 100;
+  }
+
+  const recommendPercentage = countPercentage();
+
+  return (
+    <>
```

Comment:
Why is the tag on line 22 empty?

Reply:
I tried to do React.Fragments but for some reason Airbnb style says to do empty tags instead.